native/cocos/renderer/pipeline/custom/CustomFwd.h

```diff
@@ -27,10 +27,11 @@
  * The following section is auto-generated.
  * ========================= !DO NOT CHANGE THE FOLLOWING SECTION MANUALLY! =========================
  */
-// clang-format off
 #pragma once
-#include "cocos/base/std/variant.h"
+// clang-format off
+// IWYU pragma: begin_exports
```

Comment:
Not all compilers support `IWYU pragma`.

Reply:
Shall I remove them? If vscode is used and clang-tidy is enabled with latest llvm, it will show warnings in the IDE.

Reply:
Can i know what's the warning message?

Reply:
`Included header RenderGraphFwd.h is not used directly (fix available)clangd[unused-includes](https://clangd.llvm.org/guides/include-cleaner)`

Reply:
Included header RenderGraphFwd.h is not used directly (fix available)clangd[unused-includes](https://clangd.llvm.org/guides/include-cleaner)

Reply:
`Included header RenderGraphFwd.h is not used directly (fix available)clangd[unused-includes](https://clangd.llvm.org/guides/include-cleaner)`  like this.

Reply:
Got it. I think keep it is no problem as it just fix warnings.